service/controller/app/v1/key/key.go

```diff
@@ -23,12 +37,10 @@ func ChartStatus(customResource v1alpha1.Chart) v1alpha1.ChartStatus {
 	return customResource.Status
 }
 
-func ConfigMapName(customResource v1alpha1.App) string {
-	return customResource.Spec.Config.ConfigMap.Name
-}
-
-func ConfigMapNamespace(customResource v1alpha1.App) string {
-	return customResource.Spec.Config.ConfigMap.Namespace
+// ChartConfigMapName returns the name of the configmap that stores config for
+// the chart CR that is generated for the provided app CR.
```

Comment:
This returns the name of the configmap that is generated by the configmap resource. It is used by chart-operator when calling tiller to install or update the chart.